shaas.go

```diff
@@ -113,7 +130,7 @@ func handleAny(res http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	if strings.HasPrefix(req.Header.Get("Origin"), "ws://") { // TODO: how to detect? scheme is null
+	if req.Header.Get("Upgrade") == "websocket" {
```

Comment:
This old method was `wssh`-specific. Using the `Upgrade` header works with any client.